integration/query_compat_test.go

```diff
@@ -28,14 +28,14 @@ import (
 
 // queryCompatTestCase describes query compatibility test case.
 type queryCompatTestCase struct {
-	filter        bson.D                   // required
-	sort          bson.D                   // defaults to `bson.D{{"_id", 1}}`
-	resultType    compatTestCaseResultType // defaults to nonEmptyResult
-	skipForTigris string                   // skips test for Tigris if non-empty
+	filter     bson.D                   // required
+	sort       bson.D                   // defaults to `bson.D{{"_id", 1}}`
+	resultType compatTestCaseResultType // defaults to nonEmptyResult
 }
 
 // testQueryCompat tests query compatibility test cases.
-func testQueryCompat(t *testing.T, testCases map[string]queryCompatTestCase) {
+// If skipForTigris is not empty, tigris tests are skipped.
+func testQueryCompat(t *testing.T, skipForTigris string, testCases map[string]queryCompatTestCase) {
```

Comment:
Skip whole test for tigris https://github.com/FerretDB/FerretDB/pull/1526#discussion_r1031076525